Testscripts/Windows/KERNEL-DEBUG-BASIC.ps1

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+<#
+.Synopsis
+    This script tests the debug kernel basic functionality.
+
+.Description
+    The script will install the debug kernel and boot into debug kernel,
+    then check whether have call trace
+#>
+param([String] $TestParams,
+      [object] $AllVmData)
+
+function Main {
+    param (
+        $VMname,
+        $HvServer,
+        $Ipv4,
+        $VMPort,
+        $VMUserName,
+        $VMPassword,
+        $RootDir,
+        $TestParams
+)
+
+    if ( ( $global:detectedDistro -imatch "CENTOS") -or ($global:detectedDistro -imatch "REDHAT") ) {
+        Write-LogInfo "Test on DISTRO $global:detectedDistro"
+    }
+    else {
+        Write-LogInfo "Do not support this DISTRO $global:detectedDistro."
```

Comment:
Write-LogInfo -> Write-LogErr

Reply:
Updated as Write-LogWarn.

Do we have some document to record these common usage, e.g. when return "SKIPPED", generally we should use Write-LogInfo, Write-LogErr or Write-LogWarn, then all of us can reach agreement.  Searched from whole folder, some scripts use Write-LogInfo, some use Write-LogWarn.

Sometimes not sure which style is better, similar question when using the Run-LinuxCmd and $AllVMData, Result. Do we have a document to record some common guidance for new participants? @juhlee-microsoft